### DISCUSSION TITLE
How to use?

### DISCUSSION THREAD

**Discussion by justifun** [LTmadness/ValheimBetterServerConfig#8] [Q&A] [answered]
I've installed it in the server plugin folder and launched the server like normal.

Where do i enter the commands?  in the game console?  the server cmd window ?

not sure how to use it beyond configuring the cfg file.  How can i tell is working?

**Comment by LTmadness** [comment, marked as answer]
Hi, to start of you can type 'help' in sever console, if you need help you can find me on discord as LTmadness#9705 and I can show you the ropes

![help command](https://user-images.githubusercontent.com/12985096/109787946-ad0eac00-7c06-11eb-883d-e401d5f7f1fb.PNG)
![output help](https://user-images.githubusercontent.com/12985096/109787948-ad0eac00-7c06-11eb-8c74-63ff5f7df8c8.PNG)

**Comment by papay88** [comment]
Hi, is there any possibility to do to use these commands through the console in the game? It would be very handy while playing.

**Reply by LTmadness** [reply]
you can use all of the commands using chat if the player is set as moderator/admin 
(for moderator part you must add all allowed commands into config)
